front/app/containers/HomePage/HomePage.test.tsx

```diff
@@ -1,32 +0,0 @@
-import React from 'react';
-import { render, screen } from 'utils/testUtils/rtl';
-import HomePage from '.';
-import { mockHomepageSettings } from 'services/__mocks__/homepageSettings';
-import { THomepageBannerLayout } from 'services/homepageSettings';
-
-jest.mock('services/locale');
-jest.mock('services/appConfiguration');
-jest.mock('utils/analytics');
-jest.mock('utils/cl-router/withRouter');
-jest.mock('utils/cl-router/Link');
-jest.mock('services/homepageSettings', () => ({
-  updateHomepageSettings: jest.fn(),
-}));
-
-const homepageBannerLayout: THomepageBannerLayout = 'full_width_banner_layout';
-
-jest.mock('hooks/useHomepageSettings', () =>
-  jest.fn(() => mockHomepageSettings(homepageBannerLayout))
-);
-
-describe('<HomePage />', () => {
```

Comment:
Why do you remove it? This component has quite some conditions which could be tested. And this test does test some of them. Apart from that, this test file and this simple test make the barrier for testing all possible conditions lower.

Reply:
I suppose, you remove it because it duplicates `front/app/containers/HomePage/SignedOutHeader/SignedOutHeader.test.tsx` in a way.

Maybe leaving just one `expect` instead of two would remove a part of the duplication while still testing the main thing.